samples/dictionary.sample.yml

```diff
@@ -0,0 +1,771 @@
+    - definition: beast of burden
+      sources:    [CW]
+  stem: atimw-
+
+- category: NA-3
+  FST:
+    lemma: atâhk
+  head:     atâhk
+  paradigm: NA
+  POS:      N
+  key:      atâhk
+  senses:
+    - definition: star
+      sources:    [CW, MD]
+  stem: atâhkw-
+
+- category: NA-1
+  FST:
+    lemma: acâhkos
+  head:     acâhkos
+  paradigm: NA
+  POS:      N
+  key:      acâhkos
+  senses:
+    - definition: star, little star
+      sources:    [CW, MD]
+  stem: acâhkos-
+
+- category: VAI-1
+  FST:
+    lemma: nîmiw
+  head:     nîmiw
+  paradigm: VAI
+  POS:      V
+  key:      nîmiw
+  senses:
+    - definition: s/he dances
+      sources:    [CW]
+  stem: nîmi-
+
+# nîminâniwan does not (and should not) have a LEXC entry
+- entryType:   wordform
+  fstAnalysis: nîmiw+V+AI+Ind+X
+  head:        nîminâniwan
+  lemmaSlug:   nîmiw
+  senses:
+    - definition: there is a dance, it is a time of dancing
+      sources:    [CW]
+
+- category: VAI-1
+  FST:
+    lemma: apiw
+  head:     apiw
+  paradigm: VAI
+  POS:      V
+  key:      apiw
+  senses:
+    - definition: s/he sits, s/he sits down, s/he is present
+      sources:    [CW]
+    - definition: s/he is available
+      sources:    [CW]
+    - definition: s/he is there, s/he is situated
+      sources:    [CW]
+    - definition: He sits. Also means he is at home.
+      sources:    [MD]
+  stem: api-
+
+- category: VAI-1
+  FST:
+    lemma: ay-apiw
+  head:     ay-apiw
+  paradigm: VAI
+  POS:      V
+  key:      ay-apiw
+  senses:
+    - definition: s/he sits, s/he is seated
+      sources:    [CW]
+    - definition: s/he stays seated
+      sources:    [CW]
+    - definition: He is sitting there, also he is at home.
+      sources:    [MD]
+  stem: ay-api-
+
+- category: VAI-1
+  FST:
+    lemma: nipâw
+  head:     nipâw
+  paradigm: VAI
+  POS:      V
+  key:      nipâw
+  senses:
+    - definition: s/he sleeps, s/he is asleep
+      sources:    [CW, MD]
+  stem: nipâ-
+
+- category: NDI-2
+  FST:
+    lemma: mitêýaniy
+  head:     mitêyaniy
+  paradigm: NDI
+  POS:      N
+  key:      mitêyaniy
+  senses:
+    - definition: tongue
+      sources:    [CW]
+  stem: -têýaniy-
+
+- category: VTI-1
+  FST:
+    lemma: miýâskam
+  head:     miyâskam
+  paradigm: VTI
+  POS:      V
+  key:      miyâskam1
+  senses:
+    - definition: s/he passes s.t., s/he passes s.t. walking, s/he passes around s.t., s/he bypasses s.t.
+      sources:    [CW]
+    - definition: s/he transgresses s.t.
+      sources:    [CW]
+  stem: miýâsk-
+
+- category: IPC
+  head:     miyâskam
+  paradigm: null
+  POS:      Part
+  key:      miyâskam2
+  senses:
+    - definition: past
+      sources:    [CW]
+    - definition: gone beyond (e.g. the hour)
+      sources:    [CW]
+  stem: miyâskam
+
+# This entry does not have a mapping to CW, so its exact inflectional class and paradigm are unknown.
+- category: N # QUESTION: Can there be just `N`, or does this need to be `NA` or something similar.
```

Comment:
When passed on to iDict, the presumption is that in normal circumstances we would know the inflectional category and (FST) stem, so that inflected forms could be properly recognized and paradigms generated.

In the case that we do know this information but it hasn't ended up in the FST, I could imagine that iDict still tries to generate a paradigm, but simply draws blanks on all cells.

But if we know what iDict needs to know, I think we could provide that, even if the FST side is not fully concordant.